package.json

```diff
@@ -287,6 +284,20 @@
     "ssri": "8.0.1",
     "ws": "7.4.6"
   },
+  "browserslist": {
+    "production": [
+      "chrome >= 67",
+      "edge >= 79",
+      "firefox >= 68",
+      "opera >= 54",
+      "safari >= 14"
+    ],
```

Comment:
Do we want to manually define versions? These'll get outdated pretty quickly

Reply:
we might not need this 😅 just tried a bunch of stuff until the the `bigint` bug disappeared